frontend/src/Login/Login.css

```diff
@@ -48,3 +48,24 @@ body{
 button:hover{
     cursor: pointer;
 }
+
+.role,
+.signupbutton{
+    text-align: center;
+}
+
+@media (max-width: 1000px) {
+    .logincontainer{
```

Comment:
nit, can we use camel case or pascal case for style names (https://stackoverflow.com/questions/57221878/react-classname-naming-convention)?

Reply:
Alright, I will review that.